### ISSUE TITLE
Rascil import failed using Jupyter Notebooks

### ISSUE THREAD

**Issue by Lukas113** [i4Ds/Karabo-Pipeline#142]
I got the rascil_data not found exception when I started a Jupyter Notebook started from a Jupyter server from an anaconda base environment using the venv Kernel. I've seen that if you change venvs, `karabo.util.jupyter.set_rascil_data_directory_env` is called and sets a new data directory location. However, this is not the case when I start a Notebook from the base environment and thus the import failed.

**Comment by Lukas113** [comment]
![image](https://user-images.githubusercontent.com/32157249/195135007-c06238d6-97f7-4384-9199-588b780dfb96.png)

**Comment by kenfus** [comment]
I had the same error after setting up `karabo_dev_env` on my new laptop. 

The following fixed the error for me:
```
from karabo.util.jupyter import set_rascil_data_directory_env
set_rascil_data_directory_env()
```

However, wierd is that I never enountered this problem with my old setup.

**Comment by kenfus** [comment]
I suggest adding the following two lines either to the init file in karabo, as suggested by @Lukas113. When working with `conda build`, the init files are correctly run, so I think that is a good solution.

What do you think at @deiruch ?

**Comment by deiruch** [comment]
We should fix this upstream in  #178 and introduce the changes suggested above.
